en/full-node.md

```diff
@@ -1107,10 +1107,47 @@ automatically started minimized in the task bar.
 #### Bitcoin Core Daemon {#osx-daemon}
 {:.no_toc}
 
-If you can provide instructions and screenshots for running the latest
-version of Bitcoin Core daemon on OS X Yosemite, please [open an
-issue](https://github.com/bitcoin-dot-org/bitcoin.org/issues/new) and we'll tell
-you what we need.
+The Bitcoin Core daemon (bitcoind) is not included in the .dmg file you may have downloaded to install Bitcoin-QT. Bitcoind, along with its support binaries, is instead included in the OS X .tar.gz file listed on the official Bitcoin Core download page. To download this file using Terminal, execute the following command:
+
+    curl -O https://bitcoin.org/bin/bitcoin-core-0.11.0/bitcoin-0.11.0-osx64.tar.gz
+
+{{verifyReleaseSignatures}}
+
+Extract bitcoind and its support binaries from the archive we just downloaded by running this command in Terminal:
+
+    tar -zxvf bitcoin-0.11.0-osx64.tar.gz
```

Comment:
It'd be good to remove from the instructions any hard-coded reference to a specific version of Bitcoin Core. Else we'll be in a position of having to update the instructions every time a new version is released.

Perhaps something like "In a web browser, navigate to https://bitcoin.org/en/download and click on the Mac OS X .tar.gz link to download the latest available version of the software. Next click on the downloaded .tar.gz in your downloads folder to expand the tarball into a directory of the same name."

Alternatively, @harding is there's a link available on the site for downloading the latest version of the OS X tarball? Or perhaps there's a site-wide macro whose value is the current version of Bitcoin Core?